plugins/metrics/src/main/java/org/apache/cloudstack/response/HostMetricsResponse.java

```diff
@@ -165,25 +165,25 @@ public void setNetworkWrite(final Long networkWriteKbs) {
 
     public void setCpuUsageThreshold(final String cpuUsed, final Double threshold) {
         if (cpuUsed != null && threshold != null) {
-            this.cpuThresholdExceeded = Double.valueOf(cpuUsed.replace("%", "")) > (100.0 * threshold);
+            this.cpuThresholdExceeded = Double.valueOf(cpuUsed.replace(",",".").replace("%", "")) > (100.0 * threshold);
```

Comment:
any chance of building a function to replace these characters? might be easier to maintain this code later

Reply:
This code is outdated, the function already exists.